editor/js/reaction.js

```diff
@@ -115,7 +115,9 @@ function ready() {
  */
 function listen(node) {
   addChangeHandler($(node), dirty);
-  $('.edittext').on('focus', event => selectText(event.target));
+  $('.edittext', node).on('focus', event => selectText(event.target));
```

Comment:
Note: this was improperly scoped before so that every time a new element was added to the form, a new onfocus handler was created. This led to >10 of selectText calls to be triggered every time a user clicked on a text field

Reply:
Good catch!